model/metrics/v8js-metrics.yaml

```diff
@@ -0,0 +1,69 @@
+groups:
+  - id: metric.veightjs.gc.duration
+    type: metric
+    metric_name: v8js.gc.duration
+    brief: "Garbage collection duration."
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    prefix: v8js.gc
+    attributes:
+      - ref: v8js.gc.type
+        requirement_level: recommended
```

Comment:
nit:

are there cases when it's not available? If it's always available, would it make sense to make it required?

Reply:
changed to required